indigo_app/views/works.py

```diff
@@ -241,14 +243,21 @@ class WorkOverviewView(WorkViewBase, DetailView):
 
     def get_context_data(self, **kwargs):
         context = super(WorkOverviewView, self).get_context_data(**kwargs)
+        work_tasks = Task.objects.filter(work=self.work)
 
-        context['active_tasks'] = Task.objects\
-            .filter(work=self.work)\
-            .exclude(state='done')\
+        context['active_tasks'] = work_tasks.exclude(state='done')\
             .exclude(state='cancelled')\
             .order_by('-created_at')
         context['work_timeline'] = self.get_work_timeline(self.work)
 
+        completed_tasks = work_tasks.filter(state='done')
+        context['contributors'] = User.objects\
```

Comment:
I'm going to have to think how we get Django to do the right thing here.

Reply:
I've done the reviewer and submitter separately, because it simplifies it.

Reply:
Just seen the changes, I had the same idea of counting separately then merging the tally. Guess I shouldn't have scrapped it